docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,342 @@
+   The following table lists the attributes of SoftAP:
+
+   **Table: The attributes of SoftAP**
+    <table>
+     <thead>
+      <tr>
+       <th>Attributes</th>
+       <th>Description</th>
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+       <td>SSID</td>
+       <td>Service Set Identifier (SSID)</td>
+      </tr>
+      <tr>
+       <td>Security type</td>
+       <td>None, WPA2_PSK, and WPS</td>
+      </tr>
+      <tr>
+       <td>Passphrase</td>
+       <td>Passphrase for wireless access</td>
+      </tr>
+      <tr>
+       <td>Visibility</td>
+       <td>Visibility of SSID</td>
+      </tr>
+      <tr>
+       <td>IP address</td>
+       <td>Local IP address</td>
+      </tr>
+      <tr>
+       <td>Mode</td>
+       <td>Wireless mode (B, G, A, and AD)</td>
+      </tr>
+      <tr>
+       <td>Channel</td>
+       <td>Channel</td>
+      </tr>
+     </tbody>
+    </table>
+
+- SoftAP client
+
+    You can get information about connected client such as name, IP address, MAC address, and connection time.
+
+
+## Prerequisites
+
+1. To use the SoftAP API (in [mobile](../../api/mobile/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) applications), the application must request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+     <!--Needed for configuring SoftAP settings-->
+     <privilege>http://tizen.org/privilege/softap</privilege>
+     <!--Needed for enabling and disabling SoftAP-->
+     <privilege>http://tizen.org/privilege/softap.admin</privilege>
+   </privileges>
+   ```
+2. To use the functions and data types of the SoftAP API, include the `<softap.h>` header file in your application:
+
+   ```
+   #include <softap.h>
+   ```
+
+
+## Creating SoftAP handle
+
+To create SoftAP handle:
+
+   ```
+   softap_h softap;
+   int ret = SOFTAP_ERROR_NONE;
+
+   ret = softap_create(&softap);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("softap_create failed: %d", ret);
+   ```
+
+
+
+
+## Configuring SoftAP settings
+
+To configure SoftAP settings:
+
+1. Set SSID with `softap_set_ssid()`. If you do not use this function, the device name is used as SSID:
+   ```
+   int ret = SOFTAP_ERROR_NONE;
+   softap_h softap = NULL;
+
+   ret = softap_set_ssid(softap, "Tizen");
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to set SSID");
+   ```
+
+2. Set security type and passphrase. If the passphrase is not set, random string of eight characters is used:
+   ```
+   ret = softap_set_security_type(softap, SOFTAP_SECURITY_TYPE_WPA2_PSK);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to set security type");
+
+   ret = softap_set_passphrase(softap, "1q2w3e4r!@");
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to set passphrase");
+   ```
+
+3. Set the visibility of SSID. In this example, the SoftAP client finds the SoftAP device from the Wi-Fi scan list.
```

Comment:
Suggestion:
Set the visibility of SSID. In the following example, the SoftAP client finds the SoftAP device from the Wi-Fi scan list: